libs/sdk-ui-kit/src/Dialog/ShareDialog/types.ts

```diff
@@ -1,14 +1,20 @@
-// (C) 2021-2022 GoodData Corporation
+// (C) 2021-2023 GoodData Corporation
 import { IAnalyticalBackend } from "@gooddata/sdk-backend-spi";
 import {
     IAuditableUsers,
     ObjRef,
     IAccessControlAware,
     ShareStatus,
+    IDashboardPermissions,
     IAccessGrantee,
 } from "@gooddata/sdk-model";
 import { GoodDataSdkError } from "@gooddata/sdk-ui";
 
+/**
+ * @internal
+ */
+export type CurrentUserPermissions = IDashboardPermissions;
```

Comment:
what's the added value of having this alias type for IDashboardPermissions.

Reply:
Wanted to keep the share dialog independent of the `dashboard`. All the components that consume the permissions would have to use `IDashboardPermissions` directly. But now that I think about it, permissions already have the word `dashboard` in them.

Reply:
We can probably refactor `IDashboardPermissions` to not hold the word `dashboard` in it. Let's keep this for later.